src/app/core/rpc/rpc-state/rpc-state.service.ts

```diff
@@ -22,6 +22,7 @@ export class RpcStateService extends StateService implements OnDestroy {
     super();
 
     this.register('getwalletinfo', 5000);
+    this.register('listunspent', 1000, [0]);
```

Comment:
This can in all likelihood be increased to 5000 ms (as per the getwalletinfo rpc)

Reply:
ok